src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,40 +1,89 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import cn from 'classnames';
+import { actions } from '../../features/currentTodo';
+import { getUser } from '../../api';
+import { useAppSelector } from '../../app/hooks';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
 import { Loader } from '../Loader';
 
 export const TodoModal: React.FC = () => {
+  const currentTodo = useAppSelector(state => state.currentTodo);
+  const [selectedUser, setSelectedUser] = useState<User | null>();
+  const [isError, setIsError] = useState(false);
+  const [isLoading, setIsloading] = useState(true);
+  const dispatch = useDispatch();
+  const { removeTodo } = actions;
+
+  async function loadUser() {
+    setIsloading(true);
+    try {
+      const user = await getUser((currentTodo as Todo).userId);
```

Comment:
it is better to add `if (!currentTodo) throw new Error()` to avoid such TS problems